components/dashboard/src/provider-utils.tsx

```diff
@@ -130,21 +130,30 @@ function attachMessageListener({ onSuccess, onError }: WindowMessageHandler) {
 }
 
 interface OpenOIDCStartWindowParams extends WindowMessageHandler {
-    orgSlug: string;
+    orgSlug?: string;
```

Comment:
Please add another optional `activate: boolean` property to params, ...

Reply:
Sure, what does `activate` do, and when should it be passed?

Reply:
Oh, if set, then a successful flow will activate the config?